roles/st2chatops/tasks/main.yml

```diff
@@ -0,0 +1,96 @@
+---
+- name: Assert st2chatops_hubot_adapter is specified
+  fail:
+    msg: >
+      '"st2chatops_hubot_adapter" must be supplied in roles/st2chatops/defaults/main.yml for st2chatops installation.
+      For supported hubot adapters please check: https://github.com/StackStorm/ansible-st2/blob/master/roles/st2chatops/vars/main.yml'
+  when: st2chatops_hubot_adapter not in supported_hubot_adapters
+
+- name: Assert st2chatops_hubot_adapter "{{ st2chatops_hubot_adapter|upper }}" settings are specified
+  fail:
+    msg: '"st2chatops_config" hash cannot be empty for "{{ st2chatops_hubot_adapter|upper }}" hubot adapter.'
+  when: st2chatops_config.keys() == [] and st2chatops_hubot_adapter != "shell"
+
+- name: Install latest st2chatops package
+  become: yes
+  package:
+    name: st2chatops
+    state: latest
+  when: st2chatops_version == "latest"
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Install pinned st2chatops package
+  become: yes
+  package:
+    name: st2chatops={{ st2chatops_version }}
+    state: present
+  when: st2chatops_version != "latest"
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Add st2_api_key in st2chatops.env
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '(?<=ST2_API_KEY=)(.*)$'
+    replace: '{{ st2chatops_st2_api_key }}'
+  when: st2chatops_st2_api_key != 'CHANGE-ME-PLEASE'
+  register: task_st2_api_key
+  no_log: true
+  notify: restart st2chatops
+  tags: st2chatops
+
+- name: Comment Username, Password and Auth URL, if API_KEY provided
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '{{ item }}'
+    replace: '# \1'
+  with_items:
+    - '^(export ST2_AUTH_URL.*)'
+    - '^(export ST2_AUTH_USERNAME.*)'
+    - '^(export ST2_AUTH_PASSWORD.*)'
+  when: task_st2_api_key.changed
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Add entry for "SHELL" adapter
+  become: yes
+  lineinfile:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    line: export HUBOT_ADAPTER=shell
+  when: st2chatops_hubot_adapter == "shell"
+  notify: restart st2chatops
+  register: task_shell_adapter
+  tags: st2chatops
+
+
+- name: Uncomment Hubot "{{ st2chatops_hubot_adapter|upper }}" adapter in "/opt/stackstorm/chatops/st2chatops.env"
```

Comment:
What if i specify `st2chatops_hubot_adapter=slack` in Ansible and then change it to `st2chatops_hubot_adapter=hipchat` or any other adapter later?

Will it work?